### ISSUE TITLE
Tab - Steps

### ISSUE THREAD

**Issue by dotMavriQ** [dotMavriQ/Obsitica#1]
<img width="579" alt="Image" src="https://github.com/user-attachments/assets/cad6c1ba-6677-40d9-87d1-3947650abc5c" />

**TL;DR:** 
This tab is so we can manually enter the amount of steps for each day in an easy fashion.

So we already have a working version of this that haphazardly got added to main.js instead of being properly fed through the TypeScript build process.

It just needs to be retrofitted as a proper feature of the TS stack for organizations sake, then built and confirmed to be working.

**It should:**
* Add a tab with a shoe-emoji
* Tab view is a table with two columns, _A_ lists the dates as the files inside of [journal] and _B_ what value is helt in _Properties_ for the _Key_ named _Steps_. When data is altered in a B-cell, the data should update in the corresponding files Step-Key-value.

**Comment by dotMavriQ** [comment]
I also noted that the sorting is flawed. 

Seemingly it sorts the A column by the creation date of the files, rather than in chronological order. 

Something like this should fix it:
`dates.sort((a, b) => new Date(a) - new Date(b));
`
